correios/models/posting.py

```diff
@@ -380,14 +381,12 @@ class ShippingLabel:
                         "Assinatura: __________________ Documento: _______________")
     sender_header = "DESTINATÁRIO"
     carrier_logo = os.path.join(DATADIR, "carrier_logo_bw.png")
-    receiver_data_template = ("{receiver.name!s:>.40}<br/>"
-                              "{receiver.basic_address!s:>.40}<br/>"
-                              "{receiver.complement!s:>.20} {receiver.neighborhood!s:>.20}<br/>"
+    receiver_data_template = ("{receiver.name!s:>.45}<br/>"
+                              "{basic_address!s:>.95}<br/>"
```

Comment:
Veja o impacto da quebra... se não for muito grande muda. Se for médio mantenha a implementação atual levantando uma "Deprecation Warning" e se for grande abra uma issue para fazermos isso depois.